routes/web.php

```diff
@@ -1,6 +1,8 @@
-<?php
+<<?php
```

Comment:
1つ `<` が増えてしまっているので修正をお願いします！
```suggestion
<?php
```